Libraries/CMSIS/Device/Maxim/MAX32572/Include/max32572.h

```diff
@@ -643,34 +683,33 @@ typedef enum {
 #define MXC_SPI_SS_INSTANCES (4)
 #define MXC_SPI_FIFO_DEPTH (32)
 
-#define MXC_BASE_SPI0 ((uint32_t)0x40046000UL)
-#define MXC_SPI0 ((mxc_spi_regs_t *)MXC_BASE_SPI0)
-#define MXC_BASE_SPI1 ((uint32_t)0x40047000UL)
-#define MXC_SPI1 ((mxc_spi_regs_t *)MXC_BASE_SPI1)
-#define MXC_BASE_SPI2 ((uint32_t)0x400BE000UL)
-#define MXC_SPI2 ((mxc_spi_regs_t *)MXC_BASE_SPI2)
+#define MXC_BASE_SPI0                   ((uint32_t)0x40046000UL)
+#define MXC_SPI0                        ((mxc_spi_regs_t *)MXC_BASE_SPI0)
+#define MXC_BASE_SPI1                   ((uint32_t)0x40047000UL)
+#define MXC_SPI1                        ((mxc_spi_regs_t *)MXC_BASE_SPI1)
+// SPI2 does not exist in the MAX32572 (to match instance addressing with MAX32570B)
+#define MXC_BASE_SPI3                   ((uint32_t)0x400BE000UL)
+#define MXC_SPI3                        ((mxc_spi_regs_t *)MXC_BASE_SPI3)
```

Comment:
As mentioned earlier, the ME55 is a subset of the ME13B - with a few features removed on the ME55. SPI2 peripheral location is not supported on the ME55, but the numbering is intentional to keep the SPI peripheral addresses and numbering consistent between the ME13B and ME55.